nilearn/glm/first_level/first_level.py

```diff
@@ -421,6 +421,15 @@ def fit(self, run_imgs, events=None, confounds=None,
             respective run_img. Ignored in case designs is not None.
             If string, then a path to a csv file is expected.
 
+        sample_masks : array_like, or list of array_like, optional
+            shape of array: (number of scans - number of volumes removed, )
+            Indices of retained volumes. Masks the niimgs along time/fourth
+            dimension to perform scrubbing (remove volumes with high motion)
+            and/or remove non-steady-state volumes.
+            Default=None.
+
+            .. versionadded:: 0.9.1
```

Comment:
```suggestion
            .. versionadded:: 0.9.2.dev
```
Per recommendations being added in #3225.

Reply:
Thanks for catching this!!!!